tasks/fix-cat2.yml

```diff
@@ -1271,12 +1274,56 @@
       - RHEL-07-040210
 
 - name: "MEDIUM | RHEL-07-040230 | PATCH | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
-  command: "true"
+  lineinfile:
+      dest: /var/lib/pki-kra/conf/server.xml
+      regexp: ^#?auths.revocationChecking.bufferSize
+      line: auths.revocationChecking.bufferSize=50
+  when: rhel_07_040230_audit.stat.exists == True
+  tags:
+      - cat2
+      - medium
+      - patch
+      - RHEL-07-040230
+      - pki
+
+- name: "MEDIUM | RHEL-07-040230 | PATCH | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
+  lineinfile:
+      dest: /var/lib/pki-kra/conf/server.xml
+      regexp: ^#?auths.revocationChecking.enabled
+      line: auths.revocationChecking.enabled=true
+  when: rhel_07_040230_audit.stat.exists == True
+  tags:
+      - cat2
+      - medium
+      - patch
+      - RHEL-07-040230
+      - pki
+
+- name: "MEDIUM | RHEL-07-040230 | PATCH | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
+  lineinfile:
+      dest: /var/lib/pki-kra/conf/server.xml
+      regexp: ^#?enableOCSP
+      line: enableOCSP="true"
+  when: rhel_07_040230_audit.stat.exists == True
+  tags:
+      - cat2
+      - medium
+      - patch
+      - RHEL-07-040230
+      - pki
+
+- name: "MEDIUM | RHEL-07-040230 | PATCH | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
```

Comment:
Are all these tasks for the same finding? If so, they should be combined into a single task and use a loop to iterate over them since breaking them out doesn't give any additional granularity.

```
with_items:
    - regexp: ^#?enableOCSP
      line: 'enableOCSP="true"'
      dest: /var/lib/pki-kra/conf/server.xml
```

And reference the values with `{{ item.dest }}`, `{{ item.regexp }}`, etc.